shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/metrics/MetricsServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package com.redhat.service.bridge.shard.operator.metrics;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Tag;
+
+@ApplicationScoped
+public class MetricsServiceImpl implements MetricsService {
+
+    private static final String MANAGER_REQUEST_METRICS = "http.manager.request";
+
+    @Inject
+    MeterRegistry meterRegistry;
+
+    @ConfigProperty(name = "event-bridge.sso.grant-options.password.username")
+    String username;
+
+    @Override
+    public void UpdateManagerRequestMetrics(ManagerRequestType requestType, ManagerRequestStatus status) {
+        List<Tag> tags = new ArrayList<>();
+        tags.add(Tag.of("shardId", username));
```

Comment:
`event-bridge.sso.grant-options.password.username` is not really the `shardId`. You have to inject it manually, for example the `shardId` of the `shard-1` user is https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/3b0597ebdd886d15035f3faee205f4a465e5f2f2/kustomize/overlays/minikube/keycloak/event-bridge-fm-realm-sample.json#L43 . 
You might create a new config property like `event-bridge.shard-id` and configure it accordingly

Reply:
@r00ta `username` is more readable. Could we keep it to `username` and change tag name to `shardName`.
WDYT?

Reply:
changed shardName to shardId